python/foglamp/tasks/purge/purge.py

```diff
@@ -106,9 +105,8 @@ def purge_data(self, config):
         result = self._storage.query_tbl_with_payload("streams", payload)
         last_id = result["rows"][0]["min_last_object"] if result["count"] == 1 else 0
 
-
         flag = "purge" if config['retainUnsent']['value'] == "False" else "retain"
-        if config['age']['value'] != 0:
+        if int(config['age']['value']) != 0:
```

Comment:
have we (unit) tested with (mocked) non INteger value in config for the same?! I understand config manager should handle it with type checking, but per current behavior we need to shield the code here.